kernel/api/src/main/java/org/sakaiproject/event/api/Event.java

```diff
@@ -45,6 +47,13 @@
 	 */
 	String getResource();
 
+    /**
+	 * Access the LRSStatement.
+	 * 
+	 * @return The LRSStatement.
+	 */
+	LRS_Statement getLRSStatement();
```

Comment:
Being a pedant but it would be good if this was `getLrsStatement` but if its a pain then it's ok. It's unfortunate that the class has an underscore in it 😭